devices/Vl53L0X/Vl53L0X.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Vl53L0X</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Vl53L0X</title>
+    <authors>nanoFramework project contributors</authors>
+    <owners>nanoFramework project contributors,dotnetfoundation</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <developmentDependency>false</developmentDependency>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+    <iconUrl>https://secure.gravatar.com/avatar/97d0e092247f0716db6d4b47b7d1d1ad</iconUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <description>This package includes the .NET IoT Core binding Iot.Device.Vl53L0X for .NET nanoFramework C# projects.</description>
+    <summary>Iot.Device.Vl53L0X assembly for .NET nanoFramework C# projects</summary>
+    <tags>nanoFramework C# csharp netmf netnf Iot.Device.Vl53L0X</tags>
+	  <dependencies>
+		  <dependency id="nanoFramework.CoreLibrary" version="1.10.4-preview.11" />
+		  <dependency id="nanoFramework.System.Device.Gpio" version="1.0.0-preview.40"/>
```

Comment:
Gpio needs Runtime.Events as well. Please add it. Tip: lower the version in nuget package of Gpio, it will pull it and then you can update them.